.circleci/config.yml

```diff
@@ -188,6 +188,7 @@ jobs:
       - <<: *install_node_modules
       - <<: *persist_cache
       - run: yarn markdown
+      - run: yarn markdown-link-check
```

Comment:
This seems a little misplaced here as part of validating starters.

Probably want a separate task?

Reply:
Fixed